itests/utilities/common_tx_utility.go

```diff
@@ -126,6 +151,21 @@ func Abs(x int64) int64 {
 	return x
 }
 
+func SetFromToAccounts(accountNumbers []int) (int, int) {
+	var from, to int
+	switch len(accountNumbers) {
+	case 1:
+		from = accountNumbers[0]
+	case 2:
+		from = accountNumbers[0]
+		to = accountNumbers[1]
+	default:
```

Comment:
This case should be `case 0`. In `default` case an error should be returned.

Reply:
Fix.